dotCMS/src/main/java/com/dotcms/publishing/BundlerUtil.java

```diff
@@ -211,6 +216,18 @@ public static void objectToJSON( Object obj, File f, boolean removeFirst ) {
     }
 
 
+    public static void objectToJSON( final Object obj, final OutputStream outputStream) {
+        ObjectMapper mapper = new ObjectMapper();
+
+        try {
+            mapper.writeValue(outputStream, obj);
+        } catch ( FileNotFoundException e ) {
```

Comment:
it seems you can use 
```
catch (FileNotFoundException | IOException) {
     Logger.error( PublisherUtil.class, e.getMessage(), e );
}
```

Reply:
done https://github.com/dotCMS/core/pull/20173/commits/e491040a5cc382fa8a25457ba027679b4afa86b7#diff-827653c0f7910bf99c00088d8a5123e24ca2bf4288dff44aa8b54e067f1535b5L224